daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DataProcessor.scala

```diff
@@ -718,3 +741,196 @@ class UnparseResult(dp: DataProcessor, ustate: UState)
+  var saxBlobDirectoryProperty: Path = Paths.get(System.getProperty("java.io.tmpdir"))
+  var saxBlobPrefixProperty: String = "daffodil-sax-"
+  var saxBlobSuffixProperty: String = ".blob"
+
+  private val featureMap = mutable.Map[String, Boolean](saxNamespaceFeature -> false,
+    saxNamespacePrefixFeature -> false)
+
+  override def getFeature(name: String): Boolean = {
+    if (name == saxNamespaceFeature || name == saxNamespacePrefixFeature) {
+      featureMap(name)
+    } else {
+      throw new SAXNotRecognizedException("Feature unsupported: " + name + ".\n" +
+        "Supported features are: " + featureMap.keys.mkString(", "))
+      false
+    }
+  }
+
+  override def setFeature(name: String, value: Boolean): Unit = {
+    if (name == saxNamespaceFeature || name == saxNamespacePrefixFeature) {
+      featureMap(name) = value
+    } else {
+      throw new SAXNotRecognizedException("Feature unsupported: " + name + ".\n" +
+        "Supported features are: " + featureMap.keys.mkString(", "))
+    }
+  }
+
+  override def getProperty(name: String): AnyRef = {
+    val expectedUniqueURNRegex = s"(${XMLUtils.DAFFODIL_SAX_URN_ROOT}:)(\\w+)".r
+    val prop = name match {
+      case expectedUniqueURNRegex(_, pr) if pr == "ParseResult" => saxParseResultProperty
+      case expectedUniqueURNRegex(_, bd) if bd == "BlobDirectory" => saxBlobDirectoryProperty
+      case expectedUniqueURNRegex(_, bp) if bp == "BlobPrefix" => saxBlobPrefixProperty
+      case expectedUniqueURNRegex(_, bs) if bs == "BlobSuffix" => saxBlobSuffixProperty
+      case _ =>
+        throw new SAXNotRecognizedException("Property unsupported: " + name + "." +
+          "Supported properties are: " + saxSupportedProperties.mkString(", "))
+    }
+    prop
+  }
+
+  override def setProperty(name: String, value: AnyRef): Unit = {
+    val expectedUniqueURNRegex = s"(${XMLUtils.DAFFODIL_SAX_URN_ROOT}:)(\\w+)".r
+    try {
+      name match {
+        case expectedUniqueURNRegex(_, pr) if pr == "ParseResult" => saxParseResultProperty = value
+          .asInstanceOf[ParseResult]
+        case expectedUniqueURNRegex(_, bd) if bd == "BlobDirectory" => saxBlobDirectoryProperty =
+          value.asInstanceOf[Path]
+        case expectedUniqueURNRegex(_, bp) if bp == "BlobPrefix" => saxBlobPrefixProperty = value
+          .asInstanceOf[String]
+        case expectedUniqueURNRegex(_, bs) if bs == "BlobSuffix" => saxBlobSuffixProperty = value
+          .asInstanceOf[String]
+        case _ =>
+          throw new SAXNotRecognizedException("Property unsupported: " + name + "." +
+            "Supported properties are: " + saxSupportedProperties.mkString(", "))
+      }
+    } catch {
+      case e: ClassCastException =>
+        throw new SAXNotSupportedException("Unsupported value for property: " + name + "." +
+        "Supported properties/values types are: " + saxSupportedProperties.map{
+          p => s"$p: ${p.getClass}"}.mkString(", "))
```

Comment:
I'm wondering if this is useful? This is just going to be a bunch of long URI. Maybe we just assume people can look at documentation to figure out properties? Then we don't have to update that saxSupportedProperties list each time we adda  new property?